examples/baseconfig.yaml

```diff
@@ -0,0 +1,12 @@
+paths:
+  base_path: /data/user/mwolf/data/
+  output_dir: /data/user/chaack/skyllh
+analysis:
+  name: NorthernTracks_v003p00_KDE_PDF_v003
+  datasets: IC79 IC86 MC
+  default_source:
+    ra: 77.358
+    dec: 5.693
+  livetime: 10
```

Comment:
Is it somehow possible to specify also the units in the yaml file? 10 could be 10days or 10years. The default unit in skyllh for livetime is days (which actually comes from skylab)

Reply:
in principle oyu can also use non simple types in yaml files, however I would rather add another option fo the unit, eg. `livetime_unit: "days"`

Reply:
actually, I'll just use the internal_units["time"] in the script

Reply:
The internal unit for time is seconds. I would rather like the livetime_unit config tag. It would be nice if you could automatically map the specified unit (as str) to the astropy unit. The one can specify whatever unit which astropy supports